requirements-dev.txt

```diff
@@ -7,7 +7,7 @@ mock
 nose
 peewee>2
 pep257
-psycopg2
```

Comment:
I don't think this is needed, why is this different from the normal package?

Reply:
They are renaming the package from psycopg2 to psycopg2-binary and deprecating the old name.

Reply:
github.com/psycopg/psycopg2/issues/674 this bug has links to the discussion and plan moving forward. The package specified is going to be a source only build. Which is fine for travis.